store.go

```diff
@@ -325,6 +325,9 @@ type Store interface {
 	// Layers returns a list of the currently known layers.
 	Layers() ([]Layer, error)
 
+	// LayersCleanedUp returns the layers while cleaning up duplicates before.
+	LayersCleanedUp() ([]Layer, error)
```

Comment:
Nit: `DeduplicateLayer()` might be more expressive

Reply:
Yes, I change that name :+1: